bokeh/server/configure.py

```diff
@@ -88,7 +88,7 @@ def configure_flask(config_argparse=None, config_file=None, config_dict=None):
 def register_blueprint():
     global registered
     if registered:
-        warnings.warn(
+        logging.warn(
```

Comment:
Fair enough I seem to recall some different behavior somewhere, somehow, but maybe it was some Spyder customization or something.